sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1178,7 +1177,7 @@
           {
             "name": "lockToken",
             "in": "path",
-            "description": "Lock token.",
+            "description": "The lockToken is obtained when the message was received and is provided to resolve race conditions when completing, a feedback message.",
```

Comment:
The lockToken obtained when the C2D message was received and is provided to resolve race conditions when completing a feedback message.

Reply:
changing message as per previous comments